src/askui/tools/askui/askui_controller.py

```diff
@@ -702,3 +702,26 @@ def set_display(self, displayNumber: int = 1) -> None:
             controller_v1_pbs.Request_SetActiveDisplay(displayID=displayNumber)
         )
         self._display = displayNumber
+
+    @telemetry.record_call()
+    @override
+    def run_command(self, command: str, timeout_in_milliseconds: int = 30000) -> None:
+        """
+        Execute a shell command and return the output.

```

Comment:
Mismatch between docstring and implementation: Output is not returned, so I think we should update either the docstring or implementation.